modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -382,6 +382,97 @@ definition {
 		}
 	}
 
+	@description = "LPS-165849 - Verify if terms from a related custom object (child) with a system object (parent) are replaced correctly in an email notification"
```

Comment:
`Verify terms from a 1:M related system object (parent) are replaced with the correct values in an email notification`